### DISCUSSION TITLE
Chart data usage

### DISCUSSION THREAD

**Discussion by kxMaxx** [shridhar-tl/jira-assistant#285] [Q&A]
Custom report builder is a great tool. 
Unfortunately, despite a lot of searching and trying, I haven't figured out how to use the chart component.

Question: What can I enter in "Chart data"?

Some typical examples would be great.

Thank you very much.

![Jira Assistant – ChartDara](https://user-images.githubusercontent.com/4756830/221875522-eba3de4f-d901-44aa-a189-e3baba66a563.jpg)

**Comment by shridhar-tl** [comment]
Hi @kxMaxx - Due to new restrictions added on chrome web store during late 2019, some part of report builder like multiline expressions including custom functions stopped working. There is no way to fix it at this point and even now. So I stopped developing Report Builder and left it as it is.

So though Report Builder has lots of capability for customization, it is still not a complete solution and have lots of known issues.

That being said, I don't know how far chart would be usable, and I too don't really remember how to setup things. Let me give a try and see if I could provide some examples.
